samples: protocols_serialization: server: fatal error trigger (#17065)
nrfconnect/sdk-nrf: samples/nrf_rpc/protocols_serialization/server/README.rst

* ``log_rpc`` - Enables logging over RPC.
* ``openthread`` - Enables the server part of the OpenThread RPC.

User interface
**************

Button 1:

>    * When ``log_rpc`` snippet is enabled: if pressed, triggers a fatal error (for testing the crash log feature).

Reviewer: ```suggestion
   * When the ``log_rpc`` snippet is enabled: triggers a fatal error.
     This is used for testing the crash log feature.
```
